Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3273,7 +3273,7 @@ LayoutViewportHeightExpansionFactor:
 
 LazyIframeLoadingEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
We should move to shipping for fall release, or perhaps even sooner. I'm not opposed to removing the toggle for this.

Reply:
In fairness, I didn't know it would remove the UI toggle when I made my comment. The distinction between stable and shipping wasn't clear to me. This feature has shipped so I thought shipping would be the better state. I viewed "stable" as enabled but not shipped yet.